components/dada-execute/src/heap_graph.rs

```diff
@@ -51,7 +51,7 @@ tables! {
     pub(crate) struct Tables {
         stack_frame_nodes: alloc StackFrameNode => StackFrameNodeData,
         objects: alloc ObjectNode => ObjectNodeData,
-        datas: alloc DataNode => DataNodeData,
```

Comment:
Not sure if this is a typo

Reply:
yeah, I also was in doubt about this one... data is already in plural form, from my understanding, and `datas` don't really is a word. Even so, I have done and undone this specific some times.
If it's a design choice, let me know and I change it back.

Reply:
Yeah, I think this is an intentional misspelling for naming consistency. I think it's best to leave it.

Reply:
Makes sense to me! Removed it!